h/static/scripts/annotator/guest.coffee

```diff
@@ -198,15 +201,37 @@ module.exports = class Guest extends Annotator
   _setupEditor: -> this
   _setupDocumentEvents: ->
     $(document).bind({
-      # omit the "mouseup" check
+      "touchstart": this.checkForStartSelection
       "mousedown": this.checkForStartSelection
     })
+
+    # This event is needed for iOS Safari only. 
+    if /iPhone|iPad|iPod/i.test(navigator.userAgent)
+      $(window).bind 'selectionEnd', ->
+        # reset selection timeout
+        selectionEndTimeout = null
+        # get user selection
+        selectedText = getSelectionText()
```

Comment:
This doesn't appear to do anything because `selectedText` isn't defined outside this block or used anywhere else?